puppet/modules/jenkins_job_builder/files/centos.org/ansible/provision.yml

```diff
@@ -15,12 +15,14 @@
         api_key: "{{ api_key }}"
         ssid: "{{ cico_data.results.ssid }}"
       register: cico_data
-      when: len(cico_data.results.hosts) == 0
```

Comment:
Don't you still want this here just as `when: cico_data.results.hosts|length == 0` like below?

Reply:
Never mind, realized what you were saying int he description now.